.github/workflows/turbo-ci.yml

```diff
@@ -113,7 +113,7 @@ jobs:
       - uses: dtolnay/rust-toolchain@stable
         with:
           targets: wasm32-unknown-unknown
-      - run: pnpm turbo download-keys --cache-dir=.turbo
+      - run: pnpm run compile
```

Comment:
`--cache-dir=.turbo` generally should be specified in ci workflows, or it will fail to load/save anything

Reply:
as an example, ci on this PR is currently failing